packages/plugin-core/assets/dendron-ws/tutorial/main/tutorial.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 4u6pv56mnt25d8l2wzfygu7
+title: Getting Started
+desc: ""
+updated: 1652122742329
+created: 1608051264282
+nav_order: 1.1
+---
+
+## Pre-requisites
+
+This walkthrough will help you get started with the basics of Dendron. Before you start, make sure you do the following:
+
+- Install [VS Code](https://code.visualstudio.com/download) or a VS Code compatible editor like [VSCodium](https://vscodium.com/)
```

Comment:
We shouldn't display this should we?

Reply:
since we are already copying over the tutorial, lets just tailor `tutorial.md` for vscode. can we replace the text with the following?

```
Dendron is an IDE for general knowledge.
Combine the simplicity of markdown with the power of a modern development environment. 

To begin, lets do a quick [[overview of the Dendron UI|tutorial.user-interface]]!
```